spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/controller/AppRegistryControllerTests.java

```diff
@@ -129,12 +129,15 @@ void findRegisteredApp() throws Exception {
 		mockMvc.perform(
 				post("/apps/sink/log1/3.0.0")
 						.queryParam("bootVersion", "3")
-						.param("uri", "maven://org.springframework.cloud.stream.app:log-sink-rabbit:3.0.0").accept(MediaType.APPLICATION_JSON)
+						.param("uri", "maven://org.springframework.cloud.stream.app:log-sink-rabbit:3.0.0")
```

Comment:
We still need a test here for when there is no metadata sent on registration.  I'll add it on merge.